system/include/emscripten/stack.h

```diff
@@ -7,6 +7,9 @@
 
 #pragma once
 
+#include <inttypes.h>
+#include <stddef.h>
```

Comment:
are these includes newly used?

Reply:
edit: i guess not newly used, this is just a IWYU...

Reply:
Yes.. I started using this file in isolation and it turns out they were just missing.